packages/@dcl/inspector/src/lib/data-layer/host/rpc-methods.ts

```diff
@@ -43,25 +74,22 @@ export async function initRpcMethods(
 
   // TODO: review this
   // Dump composite to the FS on every tick
-  onChanges.push(() => (dirty = true))
-  engine.addSystem(function () {
-    if (dirty) {
-      dirty = false
-
-      // TODO: hardcoded for the moment
-      composite = dumpEngineToComposite(engine, 'json')
-      // TODO: the ID should be the selected composite id name
-      // composite.id = 'main'
-
-      const mainCrdt = dumpEngineToCrdtCommands(engine)
-      fs.writeFile('main.crdt', Buffer.from(mainCrdt)).catch((err) => console.error(`Failed saving main.crdt: `, err))
-
-      compositeProvider
-        .save({ src: currentCompositeResourcePath, composite }, 'json')
-        .catch((err) => console.error(`Save composite ${currentCompositeResourcePath} fails: `, err))
-    }
+  onChanges.push(() => {
+    dirty = true
+    saveEvent.emit('change', dirty)
```

Comment:
this is not going to work for the editor since the saveEvent emitter lives on the inspector (client-side), and this code is for the data-layer (server side)